index.html

```diff
@@ -222,7 +214,7 @@ <h2>Securing the VC Data Model</h2>
     <p>
       The most specific media type (or subtype) available SHOULD be used, instead of
       more generic media types (or supertypes). For example, rather than the general
-      <code>application/sd-jwt</code>, <code>application/vc+ld+json+sd-jwt</code>
+      <code>application/sd-jwt</code>, <code>application/vc2+sd-jwt</code>
```

Comment:
```suggestion
      <code>application/sd-jwt</code>, <code>application/vc+sd-jwt</code>
```

Reply:
This was actually intentional... see https://github.com/w3c/vc-data-model/issues/1363#issuecomment-1828449697

we don't know if `application/vc` will expose media type parameters, so we have no idea if sd-jwt will need a way to signal new version of `application/vc`... hence the need to make a "new subtype".

Reply:
The vc2 thing seems like a hack to work around something.  I'd rather that we decide what the right thing to do is and do that, rather than incorporate workarounds.

Reply:
its an example, not a recommendation